templates/zerver/topic_is_muted.html

```diff
@@ -0,0 +1,9 @@
+<div id="mute_topic_container">
```

Comment:
I think we're moving to 4-space indents for most of our HTML.  I'd talk to Tim about this.  Also, @timabbott, for our relatively new backend tests for templates, will this require any kind of special test.  Since it's static (no Django interpolation), it would be nice if it could be somehow exempt from needing a specific test.  (Sorry, I'm being vague, but I don't know the exact mechanics of those tests.)